package.json

```diff
@@ -37,7 +38,6 @@
     "@small-tech/esm-tape-runner": "^2.0.0",
     "@small-tech/tap-monkey": "^1.4.0",
```

Comment:
```suggestion
```
You can remove theses deps

Also, i think we can remove `cross-env` (should not be needed for linter)